Sources/MapboxDirections/RouteLeg.swift

```diff
@@ -276,7 +294,10 @@ open class RouteLeg: Codable {
      */
     open var expectedTravelTime: TimeInterval
 
-    open var administrationRegions: [AdministrationRegion]?
+    /**
+     Containts a list of `AdministrativeRegion`'s on the current leg. `nil` value means that such data is not available.
```

Comment:
AdministrativeRegion is `:nodoc:`’d, so this property should also be hidden:

```suggestion
     :nodoc:
     The administrative regions through which the leg passes.
     
     This property is set to `nil` if no administrative region data is available.
```